src/common/utils/index.ts

```diff
@@ -113,3 +113,12 @@ export function getPercentChange(numA: number, numB: number): number | null {
  */
 export const pluralize = (num: number, singular: string, plural: string) =>
   num === 1 ? singular : plural;
+
+/**
+ * Returns true if the CAN 8.2 feature flag is enabled. The value should be
+ * updated in `.env.development` for local development and preview deployments,
+ * and in https://vercel.com/covidactnow/covid-projections/settings/environment-variables
+ */
+export function enableCan82(): boolean {
+  return process.env.REACT_APP_ENABLE_CAN_8_2 === 'true';
+}
```

Comment:
This variable doesn't add much value, except that I have seen people comparing the value as a boolean or trying to deconstruct `process.env` to read the value, both of which wouldn't work

```
// This doesn't work
const { REACT_APP_ENABLE_CAN_8_2 } = process.env;

// won't work either - env. variables are always strings
if (process.env.REACT_APP_ENABLE_CAN_8_2) {
  // ...
}

```